### DISCUSSION TITLE
How can i darken red channel?

### DISCUSSION THREAD

**Discussion by blue-devil-93** [ImageMagick/ImageMagick#5239] [Help] [answered]
Hi ,

I did buy a new flatbad scanner and it does scan the red channel a bit to light so every image has a light red tint. Is there a way to darken only the red channel by a small amount with imagemagick so the output looks more reasonable.

Kind regards blue-devil-93

**Comment by fmw42** [comment, marked as answer]
`convert image -channel r -level 0x90% +channel new_image`

change 90% as desired. If on IM 7, use magick in place of convert.

You can also do

`convert image -channel r -evaluate multiply 0.9 +channel new_image
`

Other ways are also possible

**Reply by blue-devil-93** [reply]
I will test it in a few minutes but yes that sounds similar to what i did in gimp:
https://bilderupload.org/image/6a1303231-correction-curve.png

**Reply by blue-devil-93** [reply]
convert input.jpg -channel r -evaluate multiply 0.88 +channel output.jpg  works very well. Thx for the help!!

**Comment by fmw42** [comment]
You can also use -color-matrix to adjust all channels (unix syntax)

```
convert image -color-matrix \
"r 0 0 \
0 g 0 \
0 0 b" \
result
```

where r = g = b = 1 means no change.  less than 1 is darker and higher than 1 is brighter

For Windows change \ to ^.

To darken red,

```
convert image -color-matrix \
"0.9 0 0 \
0 1 0 \
0 0 1" \
result
```

See https://imagemagick.org/script/command-line-options.php#color-matrix
